src/main/fc/settings.yaml

```diff
@@ -1514,6 +1514,30 @@ groups:
         field: ltmUpdateRate
         condition: USE_TELEMETRY_LTM
         table: ltm_rates
+      - name: sim_ground_station_number
+        field: simGroundStationNumber
```

Comment:
` condition: USE_TELEMETRY_SIM`
For all added settings. Otherwise it will be visible to F3 where it is turned off.